src/globals/scss/core/_typography.scss

```diff
@@ -139,14 +139,17 @@
   }
 
   .govuk-body-m  + .govuk-heading-l,
-  .govuk-body-s  + .govuk-heading-l {
+  .govuk-body-s  + .govuk-heading-l,
```

Comment:
Does this rule somehow relate to lists?

Reply:
The only thing that's changed is the opening parenthesis has been changed to a comma to allow the extra selectors to be added to the list…

Reply:
So it has, I must be tired 😅